examples/simple/src/comments/CommentEdit.js

```diff
@@ -31,7 +31,15 @@ const editStyles = {
 
 const CommentEdit = withStyles(editStyles)(({ classes, ...props }) => (
     <EditController {...props}>
-        {({ resource, record, redirect, save, basePath, version }) => (
+        {({
+            resource,
+            record,
+            redirect,
+            save,
+            basePath,
+            version,
+            formName,
```

Comment:
There, I don't understand. Since the form name isn't configurable in Edit and Create controllers, they should not define it, and let the `<SimpleForm>` use a default value when the prop isn't defined.

Reply:
Oups, forgot to remove this one